src/grg_sphinx_theme/theme/grg_sphinx_theme/static/styles/components/navbar.css

```diff
@@ -34,8 +34,9 @@ ol li>p:first-child, ul li>p:first-child {
   border-top: 1px solid var(--gst-color-border);
 }
 
-.nav-link i.external-icon {
+.nav-link svg.external-icon {
```

Comment:
Can you have both to be backward compatible ? Just in case we need to go back to an older version and for the older build